content.md

```diff
@@ -95,23 +95,23 @@
 
 ## 3. Registering a Member
 
-### 3.1 Start from map view seen in 2.6. Select menu-icon on the top-left of screen
+### 3.1 Start from map view(2.6.) and tap menu-icon at top-left of screen
```

Comment:
There is a space missing and an extra dot inside the parenthesis. (apply everywhere)